pkg/machine/e2e/config_init_test.go

```diff
@@ -14,6 +14,7 @@ type initMachine struct {
 	  -m, --memory uint            Memory in MiB (default 2048)
 	      --now                    Start machine now
 	      --rootful                Whether this machine should prefer rootful container execution
+		  --rosetta                Whether this machine should use Rosetta, running x86_64 Linux binaries under ARM Linux on Apple silicon
```

Comment:
What happens if this is unset? In 4.9, qemu-user-static is installed/enabled by default on the vfkit images, which is used to run x86_64 images. Does --rosetta switch between using rosetta or using qemu-user-static, or does it switch between enabling/disabling running x86_64 images?

Reply:
`--rosetta` option removed.
Use containers.conf to switch instead.
If Rosetta is turned off, qemu-user-static is used.